server/docker-base/template.gradle.properties

```diff
@@ -3,3 +3,5 @@ android.enableJetifier={{android-enable-jetifier}}
 
 # Gradle will stop resolving dependencies if android.useAndroidX is false and a dependency is using Android X
 android.useAndroidX={{android-enable-jetifier}}
+
+org.gradle.java.home=/usr/lib/jvm/java-11-openjdk-amd64
```

Comment:
Not entirely sure this is needed anymore, when using the changes to the build.gradle template, but it works now and the turnaround is a bit of a pain so...